### ISSUE TITLE
UX improvement: Warn about friend invites that are sent to friends

### ISSUE THREAD

**Issue by jolle22** [RetroShare/RetroShare#1704]
On the Retroshare windows client:
When I am in the Peoples View
And I am viewing a friends profile of a Retroshare member that I have connected to already
And I click on "Send Invite"
I want Retroshare to warn me, that the friend already is my friend instead of showing the invite email template right away.

**Comment by defnax** [comment]
yes needs feature to disable that button when you are always friends with them 

> I want Retroshare to warn me, that the friedn already is my friend instead of showing the invite email template right away.
